### PR TITLE
feat: Add Github Actions

### PR DESCRIPTION
**Pull request by DerekMaggio**
# Overview

Add Github Actions to run and teardown emulation.
Add workflow to sanity test that above Github Actions work. This will be ran every push and PR to `opentrons-emulation`

# Changelog

- Add `run_emulation.yaml` Github Action
- Add `teardown_emulation.yaml` Github Action
- Add `repo-action-validation.yaml` workflow

# Review requests

Am I doing anything really dumb in my actions?

# Risk assessment

None

### CODE REVIEW

File: .github/actions/run_emulation/action.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: 'Setup Python Inner'
```
Comment (sfoster1) [inline .github/actions/run_emulation/action.yaml:1]: old name and description maybe? probably want to use a better one

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Let's get a more descriptive name and description but the functionality looks good, is there an example of it being used somewher?

**Comment by DerekMaggio** [comment]
> Let's get a more descriptive name and description but the functionality looks good, is there an example of it being used somewher?

It's being ran in the check of this PR. I am going to add examples of using it to `opentrons`, `opentrons-modules`, and `ot3-firmware` in separate PRs.

**Review by sfoster1 — APPROVED** [review]
Awesome!

**Pull request merged**